be/src/storage/lake/lake_persistent_index.cpp

```diff
@@ -16,12 +16,55 @@
 
 #include "fs/fs_util.h"
 #include "storage/lake/filenames.h"
+#include "storage/lake/meta_file.h"
 #include "storage/lake/persistent_index_memtable.h"
 #include "storage/lake/persistent_index_sstable.h"
 #include "storage/lake/tablet_manager.h"
+#include "storage/sstable/iterator.h"
+#include "storage/sstable/merger.h"
+#include "storage/sstable/options.h"
+#include "storage/sstable/table_builder.h"
 
 namespace starrocks::lake {
 
+Status KeyValueMerger::merge(const std::string& key, const std::string& value) {
+    IndexValueWithVerPB index_value_ver;
+    if (!index_value_ver.ParseFromString(value)) {
+        return Status::InternalError("Failed to parse index value ver");
+    }
+
+    auto version = index_value_ver.versions(0);
```

Comment:
whether need to check the size before fetch the first element in the index_value_ver?